### PR TITLE
Attempt 2 - Create Character Generator HTML and CSS

### PR DESCRIPTION
**Pull request by JefferyMeverden**
# Created the Character Generator UI in HTML and CSS

# GitHub Card
#74 

# Changes

*  Created the character sheet as a landscape webpage.
*  Included a UI image for the stab block.
*  Formatted the sheet to suit the character generator.

### CODE REVIEW

File: toolkit/templates/character_generator.html [hunk 1]
```diff
@@ -3,4 +3,651 @@
 {% load static %}
 
 {% block application_pane %}
+  <div class="character-header">
+    <img src
+         {% static "Character_Header.png" %}
```
Comment (JefferyMeverden) [inline toolkit/templates/character_generator.html:8]: Ohhh, we got _pictures_ in our generator now?

File: toolkit/templates/character_generator.html [hunk 2]
```diff
@@ -3,4 +3,651 @@
...
+          </h5>
+        </div>
+        <div style="width: 50%; float: left;">
+          <h5>
+            Survival
+          </h5>
+        </div>
+      </div>
+    </div>
+  </div>
```
Comment (JefferyMeverden) [inline toolkit/templates/character_generator.html:652]: I hate CSS so much its unreal...

File: toolkit/templates/character_generator.html (hunk 1 above)
Comment (schiltz3) [reply toolkit/templates/character_generator.html:8]: 👀

Comment (schiltz3) [reply toolkit/templates/character_generator.html:8]: image

Pictures are not loading for some reason

File: toolkit/templates/character_generator.html [hunk 3]
```diff
@@ -3,4 +3,651 @@
...
+    <img src
+         {% static "Character_Header.png" %}
+         alt="Character Header"
+         style="width: 45%"/>
+  </div>
+  <div class="character-row">
+    <div class="character-name">
+      <label>
+        <h4>Character Name</h4>
+        <input class="big"
```
Comment (schiltz3) [inline toolkit/templates/character_generator.html:16]: may be worth using bootstrap's input boxes to conform with the rest of the website https://getbootstrap.com/docs/5.2/forms/input-group/#sizing

File: toolkit/templates/character_generator.html [hunk 4]
```diff
@@ -3,4 +3,651 @@
...
+        <div style="width: 50%; float: center;">
+          <h5>Proficiency Bonus</h5>
+        </div>
+      </div>
+      <div style="text-align: center;">
+        <h4>SAVING THROWS</h4>
+      </div>
+      <div class="character-panel">
+        <div style="width: 30%; float: left;">
+          <input type="checkbox"
```
Comment (schiltz3) [inline toolkit/templates/character_generator.html:176]: May be worth looking into bootstrap's check boxes and switches https://getbootstrap.com/docs/5.2/forms/checks-radios/

**Review by schiltz3 — CHANGES_REQUESTED** [review]

**Review by RonenLoundy — DISMISSED** [review]

**Review by schiltz3 — APPROVED** [review]

**Review by RonenLoundy — APPROVED** [review]

**Pull request merged**